### DISCUSSION TITLE
Not empty on multiple fields

### DISCUSSION THREAD

**Discussion by lucius100** [pocketbase/pocketbase#4173] [Q&A] [answered]
On the collection we can create record, and create new field

If I create 3 field with name [city,town,state]

can we set non empty on one of them ?

I mean on these 3 field, one of them must be non empty

All empty Not acceptable

city 
town
state

Any of them got value, acceptable

city 
town = cape
state

city = crimson
town
state

city 
town 
state = test

**Comment by ganigeorgiev** [comment, marked as answer]
There is no support for this out of the box.
You can perform your specific field checks in a hook OR as part of an API rule (_note that for the latter you'll get a generic error_). Here is an example create/update API rule (assuming that all 3 fields are `text`):

```js
// require the user to send at least one of the 3 fields
(@request.data.city != "" || @request.data.town != "" || @request.data.state != "")
```
